leakcanary-analyzer/src/main/java/com/squareup/leakcanary/AnalysisResult.java

```diff
@@ -66,6 +66,65 @@ public static AnalysisResult failure(Throwable failure, long analysisDurationMs)
   /** Total time spent analyzing the heap. */
   public final long analysisDurationMs;
 
+  /**
+   * <p>Creates a new {@link RuntimeException} with a fake stack trace that maps the leak trace.
+   *
+   * <p>Leak traces uniquely identify memory leaks, much like stack traces uniquely identify
+   * exceptions.
+   *
+   * <p>This method enables you to upload leak traces as stack traces to your preferred
+   * exception reporting tool and benefit from the grouping and counting these tools provide out
+   * of the box. This also means you can track all leaks instead of relying on individuals
+   * reporting them when they happen.
```

Comment:
Any thoughts of moving this to an optional module for those who may not need this?